<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCommandsWindow">
       <attribute name="title">
        <string>Commands Window</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTab01">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditCommand"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFileEditor">
       <attribute name="title">
        <string>File Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTab02">
        <item>
         <widget class="QTextEdit" name="textEditFile"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCommandsHistory">
       <attribute name="title">
        <string>Commands History</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTab03">
        <item>
         <widget class="QListWidget" name="listWidgetHistory"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWorkspace">
       <attribute name="title">
        <string>Workspace</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTab04">
        <item>
         <widget class="QTableWidget" name="tableWidgetWorkspace"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInteractive">
    <property name="title">
     <string>Interactive</string>
    </property>
    <addaction name="actionRun_interpreter"/>
    <addaction name="actionStop_interpreter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuInteractive"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionRun_interpreter"/>
   <addaction name="actionStop_interpreter"/>
  </widget>
  <action name="actionRun_interpreter">
   <property name="text">
    <string>Run interpreter</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionStop_interpreter">
   <property name="text">
    <string>Stop interpreter</string>
   </property>
   <property name="shortcut">
    <string>Shift+F9</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
